_source/logzio_collections/_log-sources/cloudwatch.md

```diff
@@ -511,6 +506,45 @@ To destroy the resources, use the following:
 ```bash
 terraform destroy
 ```
+</div>
+
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="test-events">
+
+#### Working with test events
+
+You can generate test events using the Logz.io Lambda test events generator and add these events to your Lambda function. This functionality is currently only available on Linux & macOS.
+
+
+<div class="tasklist">
+
+##### Generate a test event
+
+In your terminal, run the following command:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/logzio/logzio_aws_serverless/tree/master/python3/cloudwatch/test_events/test_event_generator.sh)
+```
+
+##### Add the generated test event to your Lambda function
+
+1. Select the Lambda function that you need to add the test event to.
+2. Open the **Test** tab.
+3. Select **New event**.
+4. In the **Template** field, select **CloudWatch Logs**.
+5. In the **Name** field, enter the required name for the test event.
```

Comment:
For section 5 - do you think that this sentence will make them think that there's a specific requirement for naming?
They can use whatever name they want